src/main/java/jackson/utils/Temporal.java

```diff
@@ -0,0 +1,56 @@
+package jackson.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Class used to handle LocalDate and LocalDateTime objects.
+ * This is required as we want to allow flexibility in whether the user defines
+ * a time or not, and both LocalDate and LocalDateTime strictly require no timing
+ * and timing respectively.
+ */
+public class Temporal {
+    // DateTimeFormatters for reading (first 2) from save file and user input
+    // and writing (last 2) to save file
+    private static final DateTimeFormatter dateOnly = new DateTimeFormatterBuilder()
```

Comment:
naming conventions are specific and clearly differentiates the meaning good job!